xrbk_macro/src/attribute/parsing.rs

```diff
@@ -207,11 +215,17 @@ impl ParseWithContext for ParsedAttributes {
 			}
 		}
 
-		if let Some(hide_attribute) = &hide_attribute && context_attribute.is_none() {
-			return Err(syn::Error::new(
-				hide_attribute.span(),
-				"hide attributes cannot be used without a context attribute to read the field",
-			));
+		if let Some(hide_attribute) = &hide_attribute
+			&& context_attribute.is_none()
+			&& hide_attribute
+				.hidden_traits
+				.iter()
+				.any(|r#trait| r#trait.is_ident(&format_ident!("Readable")))
+			{
+				return Err(syn::Error::new(
+					hide_attribute.span(),
+					"Cannot hide `Readable` without a context attribute to read the field",
```

Comment:
> ...actually, now that I think about it, if the hide attribute can specify the traits, then I don't think this is an error message that needs to be generated? It was from when the hide attribute applied to both reading and writing, so if you wanted to hide for writing, it would hide for reading, and you would still want a way to read it... if I remember correctly, not sure.

I think i missed something here...

Reply:
Basically, I didn't think the error was necessary.  I thought it could probably be removed.